File: C/1.Iniciante/1116.c
#include <stdio.h>

int main(void)
{
    int i, N, X, Y;
    scanf("%d", &N);
    for(i=1; i<=N; i++){
        scanf("%d %d", &X, &Y);
        if(Y==0)
            printf("divisao impossivel\n");
        else
            printf("%.1f\n", (float)X/Y);
    }
    return 0;
}

File: 1041/1116.py
qde_entradas = int(input())

for i in range(qde_entradas):
    x, y = map(int, input().split())

    if y == 0:
        print('divisao impossivel')
    else:
        divisao = x / y
        print(f'{divisao:.1f}')
        